docker-compose.user.yml

```diff
@@ -0,0 +1,28 @@
+services:
+  user-mysql:
+    image: mysql:latest
+    ports:
+      - 3306:3306
+    environment:
+      - MYSQL_ROOT_PASSWORD=secret
+    networks:
+      - user-net
+    container_name: user-mysql
+  user-micreservice:
```

Comment:
i guess it's better just user

Reply:
changed container name from 'user-micreservice'  to 'user'